server.go

```diff
@@ -116,6 +117,7 @@ func (s *Server) Serve(lis net.Listener, addr string) error {
 
 	log.Info("Accepting sessions...")
 	s.readyOnce.Do(func() { close(s.ready) })
+	defer sentry.Recover()
```

Comment:
If the `--sentry` flag is not passed and `sentry.Init` is not called, would this line work as expected?

Reply:
It will work as before.